src/plopp/static.py

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+
+from .figure import Figure
+from .io import fig_to_pngbytes
+
+
+class StaticFig(Figure):
+
+    def _repr_mimebundle_(self, include=None, exclude=None):
+        """
+        Mimebundle display representation for jupyter notebooks.
+        """
+        return {
+            'text/plain': 'Figure',
+            'image/svg+xml': fig_to_pngbytes(self._fig, form='svg').decode()
```

Comment:
Is the size of 2d figures manageable? I expect an svg is much larger than png in that case?

Reply:
yes it is, but now we `rasterize` the `pcolormesh`

Reply:
Meaning... the mesh is not vector, but all the rest is?